testsuite/helpers/echoargs.cc

```diff
@@ -0,0 +1,13 @@
+// Output command line args on stdout
+
+#include <stdio.h>
+
+int main ( int argc, char *argv[] )
+{
+    for ( int i=1; i < argc; i++ )
+    {
+	printf ( "stdout #%d: %s\n", i, argv[i] );
+    }
+   
+    return 0;
```

Comment:
Not sure about that. By convention, most (all?) Linux/Unix programs use 0 as the success exit value. This is well-known. 

Changing this to a #define that comes from some include file that is included via <stdlib.h> does not exactly add to the readability here. After all, those test helpers are meant to be concise and easy to understand, not overloaded with #define values from some deeply nested #include hierarchy.